### PR TITLE
Add TokenUtils

### PR DESCRIPTION
**Pull request by zliu41**
@ydai1124 can you review?

### CODE REVIEW

File: gobblin-azkaban/build.gradle [hunk 1]
```diff
@@ -21,12 +21,20 @@ dependencies {
   compile project(":gobblin-utility")
   if (project.hasProperty('useHadoop2')) {
     compile project(":gobblin-yarn")
+    compile externalDependency.hadoopClientCommon
+    compile externalDependency.hadoopYarnCommon
+  } else {
+    compile group: 'org.apache.hadoop', name: 'hadoop-yarn-api', version: '2.3.0'
```
Comment (ydai1124) [inline gobblin-azkaban/build.gradle]: Why is this necessary? ../build.gradle already defines hadoop version.

File: gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanTokenUtils.java [hunk 2]
```diff
@@ -0,0 +1,259 @@
...
+ *
+ * @author ziliu
+ */
+public class AzkabanTokenUtils {
+
+  private static final Logger LOG = Logger.getLogger(AzkabanTokenUtils.class);
+
+  private static final String USER_TO_PROXY = "user.to.proxy";
+  private static final String OTHER_NAMENODES = "other.namenodes";
+  private static final String KEYTAB_USER = "idpc/eat1-nertzaz01.grid.linkedin.com@GRID.LINKEDIN.COM";
```
Comment (ydai1124) [inline gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanTokenUtils.java]: Should this be extracted from `state`?

File: gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanTokenUtils.java [hunk 3]
```diff
@@ -0,0 +1,259 @@
...
+  private static final String KEYTAB_LOCATION = "idpc.azkaban.service.keytab";
```
Comment (ydai1124) [inline gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanTokenUtils.java]: same here

File: gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanTokenUtils.java [hunk 4]
```diff
@@ -0,0 +1,259 @@
...
+      renewer = new Text("azkaban mr tokens");
+    }
+
+    return renewer;
+  }
+
+  /**
+   * Get a {@link HiveConf} using the specified metastore Uri.
+   */
+  public static HiveConf getHiveConf(String metastoreUri) throws IOException {
```
Comment (ydai1124) [inline gobblin-azkaban/src/main/java/gobblin/azkaban/AzkabanTokenUtils.java]: Is this completed? What about `HiveToken`?

File: gobblin-azkaban/build.gradle (hunk 1 above)
Comment (zliu41) [inline gobblin-azkaban/build.gradle]: Otherwise it won't build with hadoop1.
I'm going to put this class in the gobblin-yarn submodule so that we don't need to do this.

**Comment by zliu41** [comment]
@ydai1124 updated 7b28049

**Comment by ydai1124** [comment]
LGTM
